game-app/game-core/src/main/java/tools/image/AutoPlacementFinder.java

```diff
@@ -269,16 +268,16 @@ private void calculate() {
     }
     textOptionPane.appendNewLine("\r\nAll Finished!");
     textOptionPane.countDown();
-    final String fileName =
-        new FileSave("Where To Save place.txt ?", "place.txt", mapFolderLocation).getPathString();
+    final Path fileName =
+        new FileSave("Where To Save place.txt ?", "place.txt", mapFolderLocation).getFile();
     if (fileName == null) {
       textOptionPane.appendNewLine("You chose not to save, Shutting down");
       textOptionPane.dispose();
       return;
     }
-    try (OutputStream os = new FileOutputStream(fileName)) {
+    try (OutputStream os = Files.newOutputStream(fileName)) {
       PointFileReaderWriter.writeOneToMany(os, placements);
```

Comment:
Note to future self: This can be simplified by refactoring the code to pass the path directly to PointFileReader writer